.releaserc

```diff
@@ -2,7 +2,16 @@
   "tagFormat": "${version}",
   "plugins": [
     [
-      "@semantic-release/commit-analyzer"
+      "@semantic-release/commit-analyzer",{
+        "preset": "angular",
```

Comment:
This is not actually what we want. This will apply both to the development branch and master. What we want is to have different rules on both branch. Master will always do a npmjs release and development will use angular rules to run a release.

two ideas for this:
- maybe we need two different semantic-release config files.
- or the semantic-release has a way to override config in the  cli